build.sbt

```diff
@@ -158,11 +152,10 @@ def versionDependent(scalaVersion: String) =
       flags ++ Seq(
         "-Xfuture",
         "-Xlint:by-name-right-associative",
-        "-Xlint:unsound-match",
+        "-Xlint:unsound-match,-infer-any",
```

Comment:
can we do a `@nowarn` where `Any` is inferred instead of disabling it across the board or would this be too pervasive?

Reply:
There is only one place, I will do it

Reply:
note that you may need scala-collection-compat for 2.12 compatibility (the annotation won't be there in 2.12 and it won't do anything with this but at least it won't break compilation)

see https://github.com/scala/scala-collection-compat/blob/v2.4.3/compat/src/main/scala-2.11_2.12/scala/annotation/nowarn.scala